app/gui/src/dashboard/layouts/UserBar.tsx

```diff
@@ -78,9 +78,9 @@ export default function UserBar(props: UserBarProps) {
   const { isOffline } = useOffline()
 
   const shouldShowUpgradeButton =
-    user.isOrganizationAdmin && user.plan !== Plan.enterprise && user.plan !== Plan.team
+    user.isOrganizationAdmin && (user.plan === Plan.free || user.plan == null)
 
-  const upgradeButtonVariant = user.plan === Plan.free ? 'primary' : 'outline'
+  const upgradeButtonVariant = user.plan === Plan.free || user.plan == null ? 'primary' : 'outline'
```

Comment:
consider:
```ts
const isUnpaidUser = user.plan === Plan.free || user.plan == null
```
and using this in both places

Reply:
I think it'd be better to normalize the user object and always define the plan field. If it's null - then free

Reply:
that works for me too!

Reply:
Addressed